sample-operators/mysql-schema/src/main/java/io/javaoperatorsdk/operator/sample/MySQLSchemaReconciler.java

```diff
@@ -40,6 +42,13 @@
 
   public MySQLSchemaReconciler() {}
 
+  @Override
+  public Optional<Object> configurationFor(DependentResourceSpec<?, ?> dependent) {
```

Comment:
We should just receive the config here, not the whole spec.

Reply:
The class is basically this class in the spec

Reply:
I hesitated here but I thought that passing the entire spec was better, in particular because I think at some point we might have a name (or some sort of qualifier) on the spec to distinguish between dependents of the same kind… Or should that be implemented using different `DependentResource` implementations? 🤔